libs/sdk-ui-dashboard/src/model/commandHandlers/dashboard/saveAsDashboardHandler.ts

```diff
@@ -179,7 +180,7 @@ function* saveAs(
                 filterContextIdentity: dashboardFilterContextIdentity(dashboardWithUser),
             }),
             layoutActions.updateWidgetIdentities(identityMapping),
-            layoutActions.clearLayoutHistory(),
+            layoutActions.clearLayoutHistory(layout),
```

Comment:
This is strange ...

Reply:
fixed by removing this.. not sure why it was complaining before..